tests/test_IP_diffusion.py

```diff
@@ -55,19 +55,14 @@ def run(dirname, vector, DG):
 
     state, f = setup_IPdiffusion(vector, DG)
 
-    if DG:
-        direction = [1,2]
-    else:
-        direction = [1]
-
-    kappa = 0.05
+    kappa = Constant(0.05)
     if vector:
         kappa = as_tensor([[kappa, 0.],[0., kappa]])
     mu = 5.
```

Comment:
I was referring to `mu`.

Though, do you plan to use a non-diagonal `kappa` at some point?  If not, why not just multiply by a scalar kappa rather than by the matrix "kappa*I"?